saleor/graphql/schema.graphql

```diff
@@ -1457,6 +1457,18 @@ type CreditCard {
   expYear: Int!
 }
 
+type CsvError {
```

Comment:
Why not just `ExportError` or `ProductExportError`?
We got `ExportEvent`, `ExportFile` and so on, and suddenly `CsvSomething` :)

Reply:
True, `ExportError` will be better.

Reply:
I'll change that.

Reply:
I remembered why I left here `CsvError`, because `csv` corresponds to our python module, where the export is, so it will be generic for export and import. Unless we want to have separate ones for export and import. @maarcingebala

Reply:
I think in the future we may need a separate type for import so `ImportError` and `ImportErrorCode`, as there may be a different set of codes there. Here I would use `ExportError` with `ExportErrorCode`. The type of export file (csv or xlsx) shouldn't matter for the error type.